src/main/java/de/mediathekview/mlib/tool/Log.java

```diff
@@ -17,19 +17,24 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-package mSearch.tool;
+package de.mediathekview.mlib.tool;
 
-import com.jidesoft.utils.SystemInfo;
-import mSearch.Config;
-import mSearch.Const;
-
-import java.io.*;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.io.StringWriter;
 import java.nio.charset.StandardCharsets;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.Date;
 import java.util.LinkedList;
 
+import com.jidesoft.utils.SystemInfo;
+
+import de.mediathekview.mlib.Config;
+import de.mediathekview.mlib.Const;
+
 public class Log {
 
     private final static String FEHLER = "Fehler(" + Const.PROGRAMMNAME + "): ";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)